packages/builders/scripts/testing/add-LEMONS.js

```diff
@@ -0,0 +1,27 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { defaultProposalBuilder as vaultProposalBuilder } from '../inter-protocol/add-collateral-core.js';
+
+/**
+ * @file
+ *     *** TESTING ONLY   ***   NOT FOR PRODUCTION USE  ***
```

Comment:
+1 to `scripts/testing`. Perhaps even `scripts/a3p`. There might be some other scripts that belong here but nbd.

That said, I think this line is too scary. There's practically zero risk that somebody is going to find `scripts/testing/add-LEMON.js` and shepherd it into production.
```suggestion
```

Reply:
Yeah, okay. I added the scary line before I renamed them. It's unneeded now.